Source/WebKit/UIProcess/WebProcessProxy.cpp

```diff
@@ -671,9 +671,10 @@ void WebProcessProxy::setThirdPartyCookieBlockingMode(ThirdPartyCookieBlockingMo
 
 Ref<WebPageProxy> WebProcessProxy::createWebPage(PageClient& pageClient, Ref<API::PageConfiguration>&& pageConfiguration)
 {
-    Ref<WebPageProxy> webPage = WebPageProxy::create(pageClient, *this, WTFMove(pageConfiguration));
+    auto isBlobPartitioningEnabled = isBlobRegistryPartitioningEnabled(*pageConfiguration->preferences());
+    Ref<WebPageProxy> webPage = WebPageProxy::create(pageClient, *this, WTFMove(pageConfiguration), isBlobPartitioningEnabled);
 
-    addExistingWebPage(webPage.get(), BeginsUsingDataStore::Yes);
+    addExistingWebPage(webPage.get(), BeginsUsingDataStore::Yes, isBlobPartitioningEnabled);
```

Comment:
Shouldn't be needed. We already pass the page and we can get the preferences from the page already.